Gaijinizer patch/Map055.txt

```diff
@@ -590,9 +598,8 @@ It is good to have arrived, but not quite at all
 > CONTEXT: Map055/events/5/pages/0/34/Dialogue/J_Animal
 > CONTEXT: Map055/events/5/pages/1/34/Dialogue/J_Animal
 > CONTEXT: Map087/events/5/pages/0/34/Dialogue/J_Animal
-That raw material is raw, and the expiration date
-is bad. So already, slaying acclaimed profit
-ignoring! If it's for you, buy it by all means!#MTLed
+Its ingredients are raw, and it's about to go off.
```

Comment:
_>**It's made up of raw ingredients, so it's about to expire.
So if you need it, please go ahead and buy some.**_